examples/map/difference_images.py

```diff
@@ -0,0 +1,63 @@
+"""
+===========================
+Plotting a difference image
+===========================
+
+How to compute and plot a difference image.
+The example uses `sunpy.map.MapSequence` to compute a difference image and then plot it.
+This basic method works for base difference or running difference. Just change whether
+you're subtracting the previous image or the first image in a sequence.
+"""
+###########################################################################
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import astropy.units as u
+from sunpy.net import Fido, attrs as a
+import sunpy.map
+
+###########################################################################
+# First we'll download a couple images and store them in a
+# `sunpy.map.MapSequence`. This could be from any channel of any imager.
+# Here, we use SDO/AIA 304 Å.
+
+instrument = a.Instrument('AIA')
+wave = a.Wavelength(30 * u.nm, 31 * u.nm)
+result = Fido.search(a.Time('2015-06-18T00:00:00', '2015-06-18T00:00:10') |
+                     a.Time('2015-06-18T01:03:30', '2015-06-18T01:03:35'), instrument, wave)
```

Comment:
```suggestion
                     a.Time('2015-06-18T01:03:30', '2015-06-18T01:03:35'),
                     instrument,
                     wave)
```

Reply:
.... weird deja vu. I could've sworn this was one of Will's suggestions too and I committed the change. Hmpf.